dockers/docker-fpm/bgpd.conf.j2

```diff
@@ -14,6 +14,7 @@ log facility local4
 ! enable password {# {{ en_passwd }} TODO: param needed #}
 {% endblock system_init %}
 !
+{% if minigraph_bgp_asn != None %}
```

Comment:
Will the following work better for Jinja2?
{% if minigraph_bgp_asn is defined %}